cohorts.md

```diff
@@ -0,0 +1,265 @@
+---
+layout: page
+title: All Open Life Science Cohorts till Date!
+image: /images/syllabus.jpg
+photos:
+  name: Niklas Morberg
+  license: CC BY-NC 2.0
+  url: https://flic.kr/p/5BXB6s
+---
+
+# Cohorts
+
+<div class='cohort-grid'>
```

Comment:
To create the cards, could you use a loop over the cohorts? It would avoid having to have to add manually always new cohort

Reply:
I have done this now. Please let me know if its ok and if there's any other thing you'd like me to add to the card so it doesn't look so bare
Thank you!